core/core-configuration-layer.el

```diff
@@ -2398,11 +2398,14 @@ depends on it."
 
 (defun configuration-layer//package-delete (pkg-name)
   "Delete package with name PKG-NAME."
-  (let ((p (cadr (assq pkg-name package-alist))))
-    ;; add force flag to ignore dependency checks in Emacs25
-    (if (not (configuration-layer//system-package-p p))
-        (package-delete p t t)
-      (message "Would have removed package %s but this is a system package so it has not been changed." pkg-name))))
+  (let ((pkg-cell (assq pkg-name package-alist)))
```

Comment:
use if-let + cadr-safe

Reply:
Done.

Reply:
I hope I got what you mean correctly. My experience in elisp is so far measured in mere days.